test/Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests/Parser/JsonStrings/SbomDocCreationJsonStrings.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Parser.JsonStrings;
+
+public static class SbomDocCreationJsonStrings
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "JSON002:Probable JSON string detected", Justification = "Need to use JSON string")]
+    public const string DocCreationJsonString =
+    @"[
```

Comment:
Pretty printing of these files is consistent. Some have different indents or no indents at al. If these are read-only test resources, it might be simpler to save them as JSON files that the tests read, or let the compiler embed them as resources, then access them via a helper method.

Reply:
The reason I chose this method was because we have to convert the json to a string anyways in order to Assert on the string equality + this way I can use regular expressions for things that might differ from run to run like creation timestamp. This is also how the unit tests are written for the Spdx22Sbom methods. 

I can format the json strings better for consistenct. But considering this, do we still want to save them as Json files instead of keeping them as strings?

Reply:
It's hard to argue with existing precedent.